.gitignore

```diff
@@ -53,8 +53,4 @@ __pycache__/
 
 #dolphin
 
-/**/__pycache__
-
-*.lock
-*.bkup
-*.log
```

Comment:
why?

Reply:
Not sure. Was added automatically. Didn't notice it.